react/components/AttachmentItem.js

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from 'react'
+import { string, number } from 'prop-types'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+
+import styles from '../productSummary.css'
+
+const AttachmentItem = ({
+  productText,
+  price
+}) => {
+  return (
+    <div className={`${styles.attachmentItemContainer} flex items-center justify-between pv1`} key={productText}>
```

Comment:
what is the prop `key` doing here? it generally should be set to the component itself (`AttachmentItem` in this case) at the point where the iteration happens